ansible/deploy_etckeeper.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: Put /etc under version control using etckeeper
+  hosts: all
+  become: True
+  roles:
+    - role: debops-contrib.etckeeper
+      tags: 'etckeeper'
```

Comment:
i thought about it, we want CVS as part of deploy_stack, however this specific role, I do want to test a bit more first, perhaps in the env I know the most or l10k.